go.mod

```diff
@@ -133,6 +133,11 @@ require (
 	oss.indeed.com/go/libtime v1.6.0
 )
 
+require (
+	github.com/vmihailenco/msgpack/v5 v5.3.5 // indirect
+	github.com/vmihailenco/tagparser/v2 v2.0.0 // indirect
+)
```

Comment:
Can we move these into the same block as the other `indirect`?

Reply:
For sure.  Don't know why tidy decided to do that.